webgpu.h

```diff
@@ -603,8 +603,17 @@ typedef enum WGPUOptionalBool {
 } WGPUOptionalBool WGPU_ENUM_ATTRIBUTE;
 
 typedef enum WGPUPopErrorScopeStatus {
+    /**
+     * `0x00000001`.
+     * The error scope stack was successfully popped and a result was reported.
+     */
     WGPUPopErrorScopeStatus_Success = 0x00000001,
     WGPUPopErrorScopeStatus_InstanceDropped = 0x00000002,
+    /**
+     * `0x00000003`.
+     * The error scope stack could not be popped, because it was empty.
+     */
+    WGPUPopErrorScopeStatus_EmptyStack = 0x00000003,
```

Comment:
I'm OK either way. I wasn't sure if we were trying to always give it value 2. (In C enums are not type safe so it's kind of nice if different enums use the same values for the same thing)

What do you mean by "override" value? We can always put InstanceDropped last but if we ever add another status value it won't be last anymore.

Reply:
Yea, I guess it probably doesn't scale... When I say override, I mean in a situation where say you have a ProcessEvent callback that is ready, but has not been resolved yet, i.e ProcessEvents hasn't been called, and you need to override.

Thinking about it more now though, I don't think it matters too much because for InstanceDropped, that usually happens via a special path anyways.

Reply:
Ah I see, statuses that overwrite other statuses. Yeah, I don't think the integer value really matters much for that, since which statuses can overwrite others is case-specific (I think InstanceDropped, and potentially Success on device loss)